app/controllers/CheckYourAnswersController.scala

```diff
@@ -35,8 +37,16 @@ class CheckYourAnswersController @Inject()(
   def onPageLoad(): Action[AnyContent] = cc.authAndGetData() {
     implicit request =>
 
+      val businessContactDetailsContactNameSummaryRow = BusinessContactDetailsSummary.rowContactName(request.userAnswers, EmptyWaypoints)
```

Comment:
Might not need waypoints as the VatRegistrationDetailsSummary does not have them